tests/rt.conf

```diff
@@ -132,7 +132,7 @@ RUN | rrfs_v1beta                                       |
 RUN | rrfs_v1nssl                                       |                            | baseline |
 RUN | rrfs_v1nssl_nohailnoccn                           |                            | baseline |
 RUN | rrfs_smoke_conus13km_hrrr_warm                    |                            | baseline |
-RUN | rrfs_smoke_conus13km_hrrr_warm_qr                 |                            |          |
+RUN | rrfs_smoke_conus13km_hrrr_warm_qr                 | - cheyenne                            |          |
```

Comment:
Can you line up the vertical bars again for the tests you've turned off? It helps to keep the rt.conf readable.

Reply:
@DeniseWorthen Oh, sure. No problem.